src/jit/arraystack.h

```diff
@@ -152,5 +161,5 @@ class ArrayStack
     int           maxIndex;
     T*            data;
     // initial allocation
-    T builtinData[builtinSize];
+    char builtinData[builtinSize * sizeof(T)];
```

Comment:
FWIW this is not the ideal solution because we're losing T's alignment. But I'm not aware of anything in the JIT that requires larger than pointer alignment and the array is pointer aligned due to the `T* data;` that precedes it.